static/usr/bin/splash-client

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -exu
+
+# Change 1 is "Initialize system state"
+msg=
+prev_msg=
+while true; do
+    sleep 0.5
+    msg=$(snap change --abs-time 1 | grep -m1 Doing |
```

Comment:
Is this suppose to show messages only on the first boot?

Reply:
Would `snap tasks --last=seed? --abs-time` be more obvious?

Should we also check `--last=become-operational`?